sdcm/nemesis.py

```diff
@@ -1101,10 +1127,36 @@ def execute_disrupt_method(self, disrupt_method):
         finally:
             self.metrics_srv.event_stop(disrupt_method_name)
 
-    def get_all_disrupt_methods(self):
-        all_disruptions = [attr[1] for attr in inspect.getmembers(self)
+    def build_list_of_disruptions_to_execute(self, nemesis_include_filter=None, nemesis_multiply_factor=1):
```

Comment:
nemesis_multiply_factor is not used

Reply:
It's explained below.
It's there for unit test purpose, but I took them out for now, till I decide what to do with them.

Reply:
I don't understand how it is going to be used in unit-test if you override it on line 1155...